### ISSUE TITLE
Text Component does not announce "disabled"

### ISSUE THREAD

**Issue by amarlette** [facebook/react-native#30937]
## Description
When using a screen reader this component does not announce that it is disabled.

This issue is covered in-depth by parent issue #30840, please check there for more details.

## React Native version:
v0.63

**Comment by lunaleaps** [comment]
If you set `accessibilityRole=button` to the component it will announce `disabled` as shown in investigation in https://github.com/facebook/react-native/issues/30947

**Comment by LilCthulu** [comment]
I would love to give this bug a try! Anything I should know to get started?

**Comment by fabriziobertoglio1987** [comment]
I found a fix for this issue see https://github.com/fabriziobertoglio1987/react-native-notes/issues/1#issuecomment-1029645388.
I will evaluate the best solution to implement in react-native.
I have a list to TO-DOs for this Pull Request here https://github.com/fabriziobertoglio1987/react-native-notes/issues/1#issuecomment-1029648248

Thanks a lot :pray:

**Comment by fabriziobertoglio1987** [comment]
@blavalla I published https://github.com/facebook/react-native/pull/33076 to fix this issue. The PR is in DRAFT. I will complete the PR in a couple of days. Thanks :pray:
